### ISSUE TITLE
Support for multiple OpenID Connect clients

### ISSUE THREAD

**Issue by jzuijlek** [micronaut-projects/micronaut-security#92]
When configuring multiple OpenID Connect clients we get the following error:

```
Error starting Micronaut server: Bean definition [io.micronaut.security.oauth2.configuration.OauthClientConfigurationProperties] could not be loaded: Error instantiating bean of type  [io.micronaut.security.oauth2.configuration.OauthClientConfigurationProperties]

Message: Multiple possible bean candidates found: [io.micronaut.security.oauth2.configuration.OauthClientConfigurationProperties$OpenIdClientConfigurationProperties$AuthorizationEndpointConfigurationProperties, io.micronaut.security.oauth2.configuration.OauthClientConfigurationProperties$OpenIdClientConfigurationProperties$AuthorizationEndpointConfigurationProperties]
Path Taken: OauthClientConfigurationProperties.setOpenid([OpenIdClientConfigurationProperties openid])
io.micronaut.context.exceptions.BeanInstantiationException: Bean definition [io.micronaut.security.oauth2.configuration.OauthClientConfigurationProperties] could not be loaded: Error instantiating bean of type  [io.micronaut.security.oauth2.configuration.OauthClientConfigurationProperties]
```

According to the documentation this should be supported: https://micronaut-projects.github.io/micronaut-security/latest/guide/#openid-configuration

**Comment by jzuijlek** [comment]
Fixed in 1.2.5

**Comment by alvarosanchez** [comment]
@jzuijlek do you mean 1.2.2?
